source/infrastructure/lib/aws-instance-scheduler-stack.ts

```diff
@@ -15,22 +15,22 @@
  *****************************************************************************/
 
 import * as cdk from 'aws-cdk-lib';
+import {Aws, RemovalPolicy} from 'aws-cdk-lib';
 import * as kms from 'aws-cdk-lib/aws-kms';
 import * as iam from 'aws-cdk-lib/aws-iam';
+import {ArnPrincipal, Effect, PolicyStatement} from 'aws-cdk-lib/aws-iam';
```

Comment:
Please either convert all `aws-iam` imports to named imports or use the namespace imports from `iam`. Multiple instances.